contributed/matching_fof_mlz_blending.ipynb

```diff
@@ -0,0 +1,805 @@
+    "    plt.scatter(coadd_data['ra'], coadd_data['dec'], s=1, rasterized=True);\n",
+    "\n",
+    "plt.xlabel('RA');\n",
+    "plt.ylabel('Dec');"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Let's choose a small RA and Dec range to do the matching so that it won't take too long!\n",
+    "ra_min, ra_max = 55.5, 56.0\n",
+    "dec_min, dec_max = -29.0, -28.5\n",
+    "\n",
+    "coord_filters = [\n",
+    "    'ra >= {}'.format(ra_min),\n",
+    "    'ra < {}'.format(ra_max),\n",
+    "    'dec >= {}'.format(dec_min),\n",
+    "    'dec < {}'.format(dec_max),\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# star, sprinkled, agn cuts for the truth catalog.\n",
+    "from GCR import GCRQuery\n",
+    "star_sprinkled_filter = [\n",
+    "    ~GCRQuery('star'),\n",
+    "    ~GCRQuery('sprinkled'),\n",
+    "    ~GCRQuery('agn')\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Lensing cuts based on Mandelbaum 2017 (arxiv 1705.06745)\n",
+    "\n",
+    "lensing_cuts = [\n",
+    "    ~GCRQuery((np.isnan, 'i_modelfit_CModel_instFlux')), # (from this and below) remove nan entries\n",
+    "    ~GCRQuery((np.isnan, 'ext_shapeHSM_HsmShapeRegauss_resolution')),\n",
+    "    ~GCRQuery((np.isnan, 'ext_shapeHSM_HsmShapeRegauss_e1')),\n",
+    "    ~GCRQuery((np.isnan, 'ext_shapeHSM_HsmShapeRegauss_e2')),\n",
+    "    GCRQuery('i_SN_cmodel >= 10'),\n",
+    "    GCRQuery('detect_isPrimary'), # (from this and below) basic flag cuts \n",
+    "    ~GCRQuery('deblend_skipped'),\n",
+    "    ~GCRQuery('base_PixelFlags_flag_edge'),\n",
+    "    ~GCRQuery('base_PixelFlags_flag_interpolatedCenter'),\n",
+    "    ~GCRQuery('base_PixelFlags_flag_saturatedCenter'),\n",
+    "    ~GCRQuery('base_PixelFlags_flag_crCenter'),\n",
+    "    ~GCRQuery('base_PixelFlags_flag_bad'),\n",
+    "    ~GCRQuery('base_PixelFlags_flag_suspectCenter'),\n",
+    "    ~GCRQuery('base_PixelFlags_flag_clipped'),\n",
+    "    ~GCRQuery('ext_shapeHSM_HsmShapeRegauss_flag'),\n",
+    "    GCRQuery('HSM_res >= 0.3'),\n",
+    "    GCRQuery('HSM_ell < 2.0'),\n",
+    "    GCRQuery('ext_shapeHSM_HsmShapeRegauss_sigma <= 0.4'),\n",
+    "    GCRQuery('mag_i_cModel < 24.5'), # FIXME: Doesnt have exinction correction\n",
+    "    GCRQuery('base_Blendedness_abs_instFlux < 10**(-0.375)'),\n",
+    "]\n",
+    "#    GCRQuery('i_SN_cmodel >= 10'), # (from this and below) cut on object properties"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Let's also define a magnitude cut\n",
+    "mag_filters = [\n",
+    "    (np.isfinite, 'mag_i'),\n",
+    "    'mag_i < 24.5',\n",
```

Comment:
I think it was separate filter to cut mag_i only if not using lensing_cut filters. I will just remove this part since we might use lensing cut all the time for the mlz&fof examples.